src/app/pointcloud/qgspointcloudlayerproperties.cpp

```diff
@@ -110,12 +131,17 @@ void QgsPointCloudLayerProperties::apply()
 
   // TODO -- move to proper widget classes!
 
+#if 0
   std::unique_ptr< QgsDummyPointCloudRenderer > renderer = qgis::make_unique< QgsDummyPointCloudRenderer >();
   renderer->setAttribute( mAttributeComboBox->currentAttribute() );
   renderer->setZMin( mMinZSpin->value() );
   renderer->setZMax( mMaxZSpin->value() );
   renderer->setColorRamp( mBtnColorRamp->colorRamp() );
   mLayer->setRenderer( renderer.release() );
+#endif
```

Comment:
remove?

Reply:
Can I leave it for now? I'll need to use this code (or similar) when adding a shade-by-attribute renderer....